app/src/main/res/values/dimens.xml

```diff
@@ -4,4 +4,26 @@
     <dimen name="list_elevation">8dp</dimen>
     <dimen name="single_line_list_item">24dp</dimen>
     <dimen name="double_line_list_item">72dp</dimen>
+    <dimen name="margin_16dp">16dp</dimen>
```

Comment:
would be better not to name it `margin_16dp` but something we could understand and reuse like `list_item_side_margin`